src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -364,7 +326,7 @@ private static string GetFixCategory(DiagnosticSeverity severity)
                 throw new NotImplementedException($"We implement {nameof(GetSuggestedActionCategoriesAsync)}. This should not be called.");
             }
 
-            private async Task<TextSpan?> GetSpanAsync(SnapshotSpan range, CancellationToken cancellationToken)
+            private async Task<TextSpan?> GetSpanAsync(ReferenceCountedDisposable<State> state, SnapshotSpan range, CancellationToken cancellationToken)
```

Comment:
why pass this around as a RCD?  why not just pass the target along?

Reply:
This form reduces the likelihood of `State` being stored for later use outside the life of this RCD instance.

Reply:
i'm skeptical.  but ok :)